src/js/components/StatusBar.js

```diff
@@ -7,7 +7,7 @@ const GROW_RATIO = 0.07;
 // Grow anything below this threshold by stealing from the biggest portion
 const MIN_SIZE = 7; // (% of statusBar width)
 
-function stealPortion(barSizes, indexesLessThanThreshold) {
+function stealPortion(barSizes, indexesLessThanThreshold, unassignedPortion) {
```

Comment:
Up to you! It was just a suggestion to make it easier for you to test and also remove clutter from the component ☺️